docker-compose.cyhy-notification.yml

```
@@ -11,4 +18,5 @@ services:
       - notification
       - --cyhy-notification-dir=/cyhy_notifications/
       - --db-creds-file=/run/secrets/database_creds.yml
+      - --csa-email-file=/run/secrets/csa_emails.yml
```

Matches my suggestion for the cli arguments.

```suggestion
      - --csa-emails-file=/run/secrets/csa_emails.yml
```